docs/SOURCES.md

```diff
@@ -1,255 +1,972 @@
+| 🇮🇱 Israel               | [IMS](#israel-meteorological-service)              | 6 days   |
+| 🇮🇹 Italy                | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+| 🇯🇪 Jersey               | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇱🇺 Luxembourg           | [MeteoLux](#meteolux)                              | 5 days   |
+| 🇲🇴 Macao                | [SMG](#serviços-meteorológicos-e-geofísicos)       | 7 days   |
+| 🇲🇨 Monaco               | [Météo-France](#météo-france)                      | 14 days  |
+| 🇲🇳 Mongolia             | [NAMEM](#namem)                                    | 5 days   |
+| 🇲🇵 Northern Mariana Is. | [NWS](#national-weather-service)                   | 7 days   |
+| 🇳🇴 Norway               | [MET Norway](#met-norway)                          | ~10 days |
+| 🇵🇭 Philippines          | [PAGASA](#pagasa)                                  | 5 days   |
+| 🇵🇹 Portugal             | [IPMA](#instituto-português-do-mar-e-da-atmosfera) | 10 days  |
+| 🇵🇷 Puerto Rico          | [NWS](#national-weather-service)                   | 7 days   |
+| 🇸🇲 San Marino           | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+| 🇸🇯 Svalbard & Jan Mayen | [MET Norway](#met-norway)                          | ~10 days |
+| 🇸🇪 Sweden               | [SMHI](#smhi)                                      | 15 days  |
+| 🇹🇼 Taiwan               | [CWA](#central-weather-administration) 🔐          | 7 days   |
+| 🇹🇷 Türkiye              | [MGM](#meteoroloji-genel-müdürlüğü)                | 5 days   |
+| 🇬🇧 United Kingdom       | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇺🇸 United States        | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇮 U.S. Virgin Is.      | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇦 Vatican City         | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+
+## Worldwide sources
+
+### Open-Meteo
+**[Open-Meteo](https://open-meteo.com/)** is a weather data provider based in Bürglen, Switzerland. It is the only free and open source weather source on this list, and probably also the most privacy-friendly. When we don’t support your national weather source, we suggest using **Open-Meteo** as your primary weather source.
+
+| Feature                        | Detail                                                                           |
+|--------------------------------|----------------------------------------------------------------------------------|
+| 🔐 **API key**                 | Not required                                                                     |
+| 🗺️ **Coverage**               | 🌐 Worldwide (some features may not be available for all locations)              |
+| ▶️ **Current observation**     | Available: most recent real-time data from the nearest weather station           |
```

Comment:
Where did you get that info? I thought it was based on the nearest 15-min ***forecast***

Reply:
I just made a template first, then copied and pasted the whole thing.
Given this, what do you suggest would be the best wording? Just "Available"?

I'm thinking from a non-technical user's perspective – how much does it matter if the headline "current" figures are from an actual observation or a forecast?

If it doesn't matter, should we draw attention to "This can be added as an 'alternative current source'"? What does "alternative current source" mean to an average user? And when would that case apply to them? I understand there was Vezza d'Oglio in Italy, where MeteoAM reports a station in Switzerland some 50km away. But it's more of an edge case, and would be very hard to explain that use case to a regular user.

Let me give this particular feature more thought.

Reply:
I don't doubt the utility of it! :) I've been playing with it from time to time myself as well.

I was (am) just pondering ways to phrase it intuitively to an average user.

Reply:
What if it's written like this instead?

> ▶️ This source can complement another weather source as a **Secondary Current Source**.

In the future, we can link to another page explaining what this means in practice.